.github/workflows/run-integration-test-dispatch.yaml

```diff
@@ -0,0 +1,87 @@
+# Workflow Dispatch to run terra-ui integration tests
+# If trying to run terra-ui tests as part of CI from other GHA, use the run-integration-tests workflow_call instead.
+name: External terraui Test
+on:
+  workflow_dispatch:
+    inputs:
+
+      ##
+      ## Configurations:
+      ##
+
+      test_url:
+        required: false
+        type: string
+        description: "URL of terraui instance to test against, required"
+      environment:
+        required: false
+        type: string
+        default: dev
+        description: "environment config to use, default: dev"
+      tests_to_run:
+        required: false
+        type: string
+        default: '[\"\"]'
+        description: "json array of tests to run, i.e. '[\"tests/find-workflow.js\",\"'tests/request-access.js'\"]' default: run all"
```

Comment:
IDK if a person will ever be running this workflow, but if so, a nicer API might be a comma separated list of test names.

Also, the empty list default and "default: run all" in the description _look_ like they might be in conflict. I assume an empty list means "run all"?

Reply:
updated input to take a simple comma seperated string, also chomps whitespaces.

Reply:
The changes I  made, turn out it breaks empty inputs, so I updated to comment that test inputs are required (for now). we can probably use the `TESTS_TO_RUN=$(yarn run jest --listTests | sed "s|$(pwd)/||")` trick that circleci uses, but it'll require a bit more refactoring to get it to work.

For now I'd like to leave it as it and we can add the "run all" feature in the future if we need to